src/System.Security.Cryptography.Xml/tests/Resources/System.Security.Cryptography.Xml.Tests.rd.xml

```diff
@@ -0,0 +1,13 @@
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+  <Library>
+
+    <!-- These types are loaded through CryptoConfig -->
+    <Type Name="System.Security.Cryptography.RSAPKCS1SignatureFormatter" Dynamic="Required Public" />
```

Comment:
We'll still need browsable metadata at minimum since Reflection is used to generate the name passed to CryptoConfig in the first place. An rd.xml on S.S.C.Xml to ensure that is probably a good idea even if we hope the reducer will be able to figure it out from the code.